guide/english/blockchain/cryptocurrency/index.md

```diff
@@ -13,6 +13,7 @@ Unlike normal currency, cryptocurrency can be exchanged as fractions. For exampl
 
 If you want to earn bitcoins through mining, it can be done through solving mathematical proof-of-work problems that validate transactions. Blockchain uses the concept of irreversible cryptographic hash function which consists of guessing a random number (usually less than a certain value) to solve the problem for transaction validation. You will require machines with high processing power to be able to solve these problems (for example Fast-Hash One or CoinTerra TerraMiner IV). Computer's with a high end Graphics Card installed (such as the Nivida GTX 1080) are also able to solve these hashes effectively. 
 
+The Downside about bitcoin is that it uses alot of energy to mine. And its not good for the enviroment.
```

Comment:
This is true. Can you please provide a citation to back up this statement?  
Please change to: `The Downside of bitcoin is that it uses alot of energy to mine, which is not good for the enviroment.`